src/components/Cluster/ClusterDetail/ClusterApps.js

```diff
@@ -240,19 +245,19 @@ class ClusterApps extends React.Component {
       name: 'cert-exporter',
       logoUrl: '/images/app_icons/cert_exporter@2x.png',
       category: 'management',
-      version: 'n/a',
+      version: null,
     },
     {
       name: 'net-exporter',
       logoUrl: '/images/app_icons/net_exporter@2x.png',
       category: 'management',
-      version: 'n/a',
+      version: null,
     },
     {
       name: 'RBAC and PSP defaults',
       logoUrl: '/images/app_icons/rbac_and_psp_defaults@2x.png',
       category: 'essentials',
-      version: 'n/a',
+      version: null,
     },
   ];
```

Comment:
I'm not so confident about this change. Does it look sane? Where can I test this manually?

Reply:
Testing on gollum. Currently there is this:

![image](https://user-images.githubusercontent.com/273727/113315481-9f376e00-930d-11eb-97a8-33db36d3a889.png)

Should there be an n/a label appearing?

Reply:
Fixed it.

![image](https://user-images.githubusercontent.com/273727/113316589-ccd0e700-930e-11eb-89e4-a04ef2d0d5ae.png)